templates/security/oval.html

```diff
@@ -0,0 +1,321 @@
+      </div>
+    </section>
+
+    <section class="p-strip--light">
+      <div class="row">
+        <div class="col-7">
+          <h2>
+            Ubuntu OVAL data parameters
+          </h2>
+          <table>
+            <thead>
+              <tr>
+                <th scope="col" style="width: 33%;">
+                  Parameter
+                </th>
+                <th scope="col">
+                  Description
+                </th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  CVE_ID
+                </td>
+                <td>
+                  CVE number as reported by MITRE
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  USN
+                </td>
+                <td>
+                  Corresponding Ubuntu Security Notice
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  Description
+                </td>
+                <td>
+                  A short description of the security risk addressed
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  Severity
+                </td>
+                <td>
+                  CVE or USN severity as defined by the Ubuntu Security team
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  Affected Platform
+                </td>
+                <td>
+                  Affected Ubuntu release(s), incl ESM
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  Title
+                </td>
+                <td>
+                  CVE number, affected Ubuntu release(s), and Severity
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  Public date
+                </td>
+                <td>
+                  The date on which a CVE was publicly announced
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  Public date of USN
+                </td>
+                <td>
+                  The date on which a security patch was announced or provided
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  Reference
+                </td>
+                <td>
+                  Links to more information about the issue
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  BugReport
+                </td>
+                <td>
+                  Link to bugreport about the issue
+                </td>
+              </tr>
+            </tbody>
+          </table>
+          <p>
+            <small>
+              Note: The above parameters are included in the OVAL xml file, but not all are shown in the resulting generated OpenSCAP report.
```

Comment:
Should this be a notification pattern? If you want to keep it subtle then you could replace "Note:" with Vanilla's p-icon--information icon.

Reply:
updated